### ISSUE TITLE
Spell Timers giving error

### ISSUE THREAD

**Issue by Deezelgurl** [DeadlyBossMods/DeadlyBossMods#220]
DBM is throwing an error regarding Spell Timers causing DBM not to function unless you disable the Spell Timers node. 
![DBM Spell Timers 2020-05-31 095722](https://user-images.githubusercontent.com/26132625/83358029-4aaa1780-a325-11ea-817c-51390321bfe9.png)

**Comment by QartemisT** [comment]
[![Watch the video](https://img.youtube.com/vi/PdtJyKyZB4k/maxresdefault.jpg)](https://youtu.be/PdtJyKyZB4k)

**Comment by Deezelgurl** [comment]
DBM is up to date as of 5/29 and there are no new updates. And I cannot find a specific update for only the spell timers aspect of the add on. No need to be a jerk about it.

**Comment by QartemisT** [comment]
Remove DBM-SpellTimers, as the message says.

**Comment by Deezelgurl** [comment]
I will remove it. Figured you would at least want to know about the problem. Lesson learned.

**Comment by QartemisT** [comment]
See the following announcement that has been posted on twitter and on our discord:
https://twitter.com/deadlybossmods/status/1266867979465240581

**Comment by MysticalOS** [comment]
Thanks for being helpful. Generally speaking if it throws an actual script error that's a bug. if it prints a localized message like that, it means a human had to have typoed that out so it's intended.

**Comment by Deezelgurl** [comment]
Until I posted here, I never would have known that the two portions were no longer being supported. I don't use Twitter and I have no reason to be a Discord member as I've hardly had any issues. Maybe update the error message to reflect that it's no longer being supported instead of just saying that it "needs an update"? That's the main reason I reached out because I could not locate an update. I am very capable of reading, just don't use the two platforms you posted the notification on. Thanks.
